frontend/pages/projects/_id/affective-annotation/index.vue

```diff
@@ -337,13 +337,16 @@ export default {
 
   async fetch() {
     this.isProjectAdmin = await this.$services.member.isProjectAdmin(this.projectId)
-    await this.setProjectData()
-    await this.setDoc()
-    await this.setHasCheckedPreviousDoc()
-    this.$nextTick(()=> {
-      this.setArticleData()
-      this.loadLabels()
-    })
+    const isAllowed = this.isAllowedToAnnotate()
+    if (isAllowed) {
+      await this.setProjectData()
+      await this.setDoc()
+      await this.checkHasCheckedPreviousDoc()
+      this.$nextTick(()=> {
+        this.setArticleData()
+        this.loadLabels()
+      })
+    }
```

Comment:
I think user should be notified that they can't annotate this project now, wdyt?

Reply:
hello, I have added the warning message:
<img width="960" alt="ssasdhgvjhdsv" src="https://user-images.githubusercontent.com/64476430/201183603-df7f6bda-d857-4686-9189-9b89f1cb47fa.PNG">

Reply:
also, for the time being I have re-enabled the persistedstate plugin. since now only `userState` module is saved into local storage, i hope it will not cause any conflict/issue. would you please check it? 😓

Reply:
Hello, I got some comments I failed to notice previously, can you check it?